src/main/scala/za/co/absa/abris/avro/registry/SchemaSubject.scala

```diff
@@ -30,31 +30,27 @@ class SchemaSubject(val asString: String) {
 }
 
 object SchemaSubject{
-  private val TOPIC_NAME_STRATEGY = new TopicNameStrategy()
-  private val RECORD_NAME_STRATEGY = new RecordNameStrategy()
-  private val TOPIC_RECORD_NAME_STRATEGY = new TopicRecordNameStrategy()
-
 
   def usingTopicNameStrategy(
     topicName: String,
     isKey: Boolean = false
   ): SchemaSubject = {
-    val dummySchema = createDummySchema("name", "namespace")
-    new SchemaSubject(TOPIC_NAME_STRATEGY.subjectName(topicName, isKey, dummySchema))
+    val suffix = if (isKey) "-key" else "-value"
```

Comment:
Maybe add a comment here, with a reference to the `TopicNameStrategy` implementation from confluent, just so it's transparent where the `-key` and `-value` convention comes from.

Reply:
Is the link on top of SchemaSubject class enough, or should there be a special comment here as well?
The link is hidden in the diff, so I don't know if you saw it.

Reply:
Ah ok, then it's fine. No, I didn't see it